.github/CODEOWNERS

```diff
@@ -477,6 +477,7 @@
 
 # Learner group project.
 /core/domain/learner_group*.py @aks681 @paprajapati9
+/core/templates/domain/learner_group/ @aks681
```

Comment:
Can you change this to oppia/lace-front end-reviewers? Also, edit the line above.

Reply:
done